### DISCUSSION TITLE
Web API docs have outdated information in zh-CN. Up-to-date translation is needed

### DISCUSSION THREAD

**Discussion by yangmingli08** [mdn/mdn-community#278] [Localization] [answered]
Many translations of the web API docs in zh-CN are outdated. It may mislead some the programmers like my colleagues who don't speak English and may ruin the reputation of this website. I speak both languages and am happy to be a volunteer to improve localization. Is there a community doing this in China?
BTW not zh-TW. The language is slightly different from zh-CN.

**Comment by yin1999** [comment, marked as answer]
Hi @yangmingli08. Thanks for the interests. We welcome contributors from all over the world.

If you want to help with improving l10n-zh documents. You can create PR or report issues in [mdn/translated-content](https://github.com/mdn/translated-content). But before starting out, you can read some documents about [how to Contribute](https://developer.mozilla.org/zh-CN/docs/MDN/Contribute):

- [Contributing to MDN](https://developer.mozilla.org/zh-CN/docs/MDN/Contribute) - show how can you help us.
- [Page structures](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures) - the page structures of MDN Web Docs.
- [translation guidelines of zh-CN](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md) - what we should take care of when contributing to zh-CN (zh-TW as [here](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md))

See also: <https://github.com/mdn/translated-content/discussions/7749#discussioncomment-3418895>

**Reply by yangmingli08** [reply]
OK Thanks

**Comment by pransh15** [comment]
Closing since this has been answered.
